### DISCUSSION TITLE
`license_url` not used on Anaconda.org

### DISCUSSION THREAD

**Discussion by jakirkham** [conda/infra#739] [General]
Despite having `license_url` in the `libcublas` recipe and packages (for [example]( https://github.com/conda-forge/libcublas-feedstock/blob/7ede7517d0d09d649f582eed3aaa7f0a2bb29997/recipe/meta.yaml#L62 )), Anaconda.org appears not to be using it and instead generating a link to a broken page. The other links work.

![image](https://user-images.githubusercontent.com/3019665/229248362-eb1a886f-663c-4aeb-8e91-42d1e6687d76.png)

We have seen this issue before ( https://github.com/conda-forge/cudatoolkit-feedstock/issues/2 ). Though thought this issue was fixed there.

Could someone please take a look?

Thanks in advance for your help! 🙏

**Comment by jakirkham** [comment]
Should add this seems similar to this issue ( https://github.com/conda/infrastructure/discussions/649 )

**Comment by leofang** [comment]
Identical issues:
- https://github.com/conda-forge/cuquantum-feedstock/issues/3
- https://github.com/conda-forge/cusparselt-feedstock/issues/14
- https://github.com/conda-forge/cuda-python-feedstock/issues/7

**Comment by jakirkham** [comment]
@jessewiles could you please help us here?

**Reply by jessewiles** [reply]
I have replicated this issue and can confirm it's a defect in the server code.

**Reply by jakirkham** [reply]
Thanks Jesse! 🙏

Am curious do you have a sense of what it would take to fix?

**Reply by jakirkham** [reply]
IIUC there has been a fix deployed (though please correct me if I misunderstood)

Do we know when it will be available?

**Reply by jakirkham** [reply]
^ @jessewiles ?

**Reply by jessewiles** [reply]
The release is going through QA now. I would it expect it to be deployed next week. I'll keep you updated @jakirkham.

**Reply by jakirkham** [reply]
Thanks @jessewiles! 🙏

Have you heard any updates on this? Did it go out already?

**Reply by jakirkham** [reply]
In talking with @kathatherine in the Conda Community meeting earlier, it sounds like this release is still in QA, but is planned to ship this week

If someone could please update this thread when it goes out, that would be greatly appreciated 🙏

**Reply by jakirkham** [reply]
Asked at the community meeting this week, it sounds like Anaconda will check the status and someone will report back here

**Reply by rasquith** [reply]
Hi @jakirkham, thanks for following up with us--sorry for the brief technical difficulties. It looks like we just released a fix yesterday evening. The link looks like it's working for me, but please confirm that it looks okay.

**Reply by jakirkham** [reply]
No worries. Perhaps there are some HackMD permissions that need tweaking. Maybe @tnabtaf can help ensure the agenda works for you 🙂 

<hr>

Indeed this looks really good 😄 Thanks @rasquith! 🙏

Really happy to see that existing `license_url`s are being picked up without needing to rebuild packages 🎉 

Tried a few links and they seemed to work

@leofang can you please also check?

@adibbley when you are around, please also let us know whether this is working for you

**Reply by leofang** [reply]
Thanks @jakirkham @rasquith for following up. I see all but one packages are fixed: cuda-python's license redirect is still incorrect. https://anaconda.org/conda-forge/cuda-python

**Reply by rasquith** [reply]
Okay @leofang we're looking into it.

**Reply by rasquith** [reply]
We recommend that you update the package metadata.

**Reply by jakirkham** [reply]
Hi @rasquith , thanks for taking a look.

The [`license_url` is already specified in the recipe]( https://github.com/conda-forge/cuda-python-feedstock/blame/8f1cd6b6123e45dd180957f4bd4a7c711ca118c0/recipe/meta.yaml#L73 ) and that has been the case for quite some time

Could you please take another look?

**Reply by rasquith** [reply]
Gotcha. Thanks, will definitely take a look.

**Reply by leofang** [reply]
Hi @rasquith any update?

**Comment by jakirkham** [comment]
Looking at this recently, it appears that the wrong link is no longer being added to the "License" field, which is an improvement.

However the right link is not being added, which causes this to be rendered as text:

<img width="1840" alt="Screenshot 2023-10-23 at 4 04 45 PM" src="https://github.com/conda/infrastructure/assets/3019665/94a4756d-9d40-4f09-b5df-1f9dd6881f90">

Note that Anaconda.org is aware of what the "License URL" should be. It just doesn't seem to use it for some reason:

<img width="1840" alt="Screenshot 2023-10-23 at 4 06 50 PM" src="https://github.com/conda/infrastructure/assets/3019665/70dbad6e-2a14-4ac9-a227-a3956ea49a8e">

**Reply by rasquith** [reply]
Thanks for bringing this to our attention @jakirkham. We're creating a ticket for "Everything around licenses works as it should" to address this issue and see if we can get ahead of any remaining weird behavior. We'll look to implement any changes after a delay for a planned infrastructure update. If you identify any other related issues, feel free to drop them in this discussion.

**Comment by jakirkham** [comment]
cc @aterrel (for awareness)

**Comment by jakirkham** [comment]
cc @jezdez

**Comment by jakirkham** [comment]
Am curious what the status on this issue is

Seeing the same state we saw a year ago: https://github.com/conda/infrastructure/discussions/739#discussioncomment-7363975
